code/game/objects/structures/signs.dm

```diff
@@ -466,9 +467,14 @@
 		claimant = user.real_name
 	..()
 
+/obj/item/sign/medipolma/attackby(obj/item/W as obj, mob/user as mob)
+	if(istype(W, /obj/item/tape_roll))
+		user.visible_message(SPAN_WARNING("[user] attaches [W] to the [src]."), SPAN_WARNING("You attach some [W] to the [src]."))
+	return ..()
+
 /obj/item/sign/medipolma/examine(mob/user)
 	. = ..()
 	if(claimant)
-		to_chat(user,"This one belongs to Dr.[claimant], MD.")
+		to_chat(user,"This one belongs to Dr. [claimant], Chief Medical Officer.")
```

Comment:
You don't understand what MD means don't you

Reply:
medical doctor